part1/unicafe/src/App.jsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react'
+
+// a proper place to define a component
+const Statistics = ({presentStatistics, good, neutral, bad}) => {
+  // ...
+
+  const total = {good} + {neutral} + {bad}
```

Comment:
```suggestion
  const total = good + neutral + bad
```

I think you are mixing JS and JSX. JSX is used in the braces of the return statement and requires the braces to access variables that are used elsewhere in the file/project. https://webtips.dev/javascript-vs-jsx